build/tasks/aria-supported.js

```diff
@@ -0,0 +1,59 @@
+/*eslint-env node */
+'use strict';
+
+const aQ = require('aria-query');
+const { roles, aria: props } = require('aria-query');
+
+module.exports = function (grunt) {
+	grunt.registerMultiTask(
+		'aria-supported',
+		'Task for generating a diff of supported aria-roles.',
+		function () {
+			const entry = this.data.entry;
+			const destFile = this.data.destFile;
+			const listType = this.data.listType.toLowerCase(); // enum = 'supported', 'unsupported', 'all'
+
+			const axe = require('../../axe');
+
+			const ariaKeys = Array.from(props).map(([key]) => key)
+			const roleAriaKeys = Array.from(roles)
+				.reduce((out, [name, rule]) => {
+					return [...out, ...Object.keys(rule.props)]
+				}, []);
+			const aQaria = new Set(axe.utils.uniqueArray(roleAriaKeys, ariaKeys));
+
+			const getDiff = (base, subject) => {
+				return [...[...new Map([...base.entries()].sort())]]
```

Comment:
Creative... but I have no idea what this does, and likely neither will you three weeks from now ;-)

Reply:
It's creating a sorted list of unique elements.

Could probably just be `Array.from(base.entries()).sort()` tho 😉